.github/workflows/publish-release-artifacts.yml

```diff
@@ -39,21 +44,9 @@ jobs:
           echo "Project Version: $PROJECT_VERSION"
           echo "Release Tag: $RELEASE_TAG"
           [ "$PROJECT_VERSION" = "$RELEASE_TAG" ] || exit 1
-  publish-to-github-release:
-    # only run if `check-tag` completes successfully
-    needs: check-tag
-    runs-on: ubuntu-latest
-    permissions:
-      contents: write
-    steps:
-        # TODO: replace with artifact upload/download -- make sure there's no race condition with other builds also
-        # uploading an artifact.
-      - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v3.6.0
-        with:
-          submodules: recursive
       - uses: gradle/gradle-build-action@8baac4c8ef753599f92eeb509c246d09d6250fa6 # v3.3.0
         with:
-          arguments: build cyclonedxBom
+          arguments: build cyclonedxBom publishToSonatype closeAndReleaseSonatypeStagingRepository
```

Comment:
This is the main difference, right? If I understand correctly it means we won't have to log into the web UI and manually click through to release the staged repo, right?

Reply:
Yes—that's correct.